src/Features/Traits/HttpResponse.php

```diff
@@ -0,0 +1,252 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Blumilk\BLT\Features\Traits;
+
+use Behat\Gherkin\Node\TableNode;
+use PHPUnit\Framework\Assert;
+use Symfony\Component\CssSelector\CssSelectorConverter;
+use Symfony\Component\DomCrawler\Crawler;
+use Symfony\Component\HttpFoundation\Response;
+
+trait HttpResponse
+{
+    protected Response $response;
+
+    /**
+     * @When a request is sent
```

Comment:
```suggestion
     * @When a response is received
```
Maybe will be more accurate for handling responses